packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -713,7 +728,15 @@ class EditableText extends StatefulWidget {
        assert(rendererIgnoresPointer != null),
        assert(scrollPadding != null),
        assert(dragStartBehavior != null),
+       assert(
+          (contentInsertionMimeTypes.isNotEmpty && onContentInserted != null) ||
+          contentInsertionMimeTypes.isEmpty,
```

Comment:
We should revisit this assertion when a decision is made on the default value discussed below.

Reply:
Outdated as the assertion is no longer required